src/enclave/httpendpoint.h

```diff
@@ -95,7 +117,29 @@ namespace enclave
       session_id(session_id)
     {}
 
+    struct SendCbMsg
```

Comment:
This is the same type as `send_raw_msg` in `tlsendpoint.h`. Could we remove this one?